### PR TITLE
fix: modal interaction

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Initialize all modals hidden
- Add close user menu when click in upgrade plan button

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -23,8 +23,15 @@ $("body").on('click', '#close-menu', function() {
   $('.header-main').removeClass('open');
 });
 
+$("body").on('click', '.close-user--menu', function() {
+  $('.user-menu').removeClass('open');
```
Comment (andresmoschini) [inline assets/js/main.js]: It closes the menu when the user clicks on, for example, "COMPRAR". But should you also close the menu when the user clicks outside the menu?

image

File: assets/js/main.js [hunk 2]
```diff
@@ -23,8 +23,15 @@ $("body").on('click', '#close-menu', function() {
...
+});
+
 // modals
 var $delayFade = 500;
+//Initialize all modals hidden
+$("#modal-large, #modal-medium, #modal-small").hide();
```
Comment (andresmoschini) [inline assets/js/main.js]: 👍

File: assets/js/main.js (hunk 1 above)
Comment (fcoronelmakingsense) [reply assets/js/main.js]: This is another thing we need to add. I add this for the current modal we need

**Comment by cbernat** [comment]
👍

**Pull request merged**